README.rst

```diff
@@ -14,7 +14,7 @@
   :target: CODE_OF_CONDUCT.md
 
 
-**pyActigraphy**
+**pyActigraphy** 
```

Comment:
Is it needed? I do not see any formatting issue on GitHub.com, nor on GitHub.io.

Reply:
This is not required. I was just playing with GitHub desktop.